src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj

```diff
@@ -76,6 +78,15 @@
     <Compile Include="..\MongoDB.Shared\MaxTimeHelper.cs" Link="Core\Operations\MaxTimeHelper.cs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Include="snappy64.dll">
```

Comment:
Also, these libraries can be placed in a new folder like: `Native` or so.
@rstam

Reply:
I think the location for this file should be:

    ...\src\MongoDB.Driver.Core\Core\Compression\Snappy\lib\win

And `win` should have `linux` and `macos` sibling directories.